test/emqx_protocol_SUITE.erl

```diff
@@ -17,72 +17,67 @@
 -module(emqx_protocol_SUITE).
 
 -compile(export_all).
+
 -compile(nowarn_export_all).
 
+-define(APP, ?PID).
+
+-define(PID, emqx).
+
 -include_lib("eunit/include/eunit.hrl").
 
 -include_lib("common_test/include/ct.hrl").
 
 -include("emqx_mqtt.hrl").
 
--define(TOPICS, [<<"TopicA">>, <<"TopicA/B">>, <<"Topic/C">>, <<"TopicA/C">>,
-                 <<"/TopicA">>]).
+-define(WILD_TOPICS,
```

Comment:
These topic filters has been tested in other test suites, please don't add duplicated test suites.